app/jobs/export_job.rb

```diff
@@ -5,34 +5,56 @@ class ExportJob < ApplicationJob
   queue_as :exports
   include ImportExport
 
-  # Limits number of Patient records to be considered for a single exported file to handle maximum file size limit.
-  # Adds additional files as needed if exceeds batch size.
-  OUTER_BATCH_SIZE = ENV['EXPORT_OUTER_BATCH_SIZE']&.to_i || 10_000
-
   # Inner batch size limits number of Patient records details help in memory at once before writing to file.
-  INNER_BATCH_SIZE = ENV['EXPORT_INNER_BATCH_SIZE']&.to_i || 500
+  INNER_BATCH_SIZE = ENV.fetch('EXPORT_INNER_BATCH_SIZE', 500).to_i unless const_defined?(:INNER_BATCH_SIZE)
 
   def perform(config)
     # Get user in order to query viewable patients
     user = User.find_by(id: config[:user_id])
     return if user.nil?
 
     # Delete any existing downloads of this type
-    user.downloads.where(export_type: config[:export_type]).delete_all
+    user.downloads.where(export_type: config[:export_type]).destroy_all
```

Comment:
can we add this comment (same one from purge job) here as well:
```
# Destroy all must be called on the downloads because the after destroy callback must be executed to remove the blobs from object storage
```